package.json

```diff
@@ -32,20 +33,23 @@
   "license": "MIT",
   "homepage": "https://github.com/herbsjs/herbs2gql#readme",
   "dependencies": {
-    "gotu": "^1.0.6"
+    "buchu": "^1.0.7",
+    "gotu": "^1.0.6",
+    "lodash": "^4.17.20"
```

Comment:
Why we need lodash here? Can be inside devdependencies?

Reply:
Hi @jhomarolo !
I Includes error scenarios for unit testing. Can you see please?

Reply:
Hi @jhomarolo !
Lodash is used to convert text to `camecasel`. In the future we must implement this native function in **suma**.